components/backup-stream/src/endpoint.rs

```diff
@@ -111,14 +114,14 @@ where
         concurrency_manager: ConcurrencyManager,
     ) -> Self {
         crate::metrics::STREAM_ENABLED.inc();
-        let pool = create_tokio_runtime(config.io_threads, "backup-stream")
+        let pool = create_tokio_runtime(4, "backup-stream")
```

Comment:
So it is a fixed number? Does 4 works well on different Cloud tiers like 2c/4c/8c/16c dedicated clusters?

Reply:
Good advice~, have changed it as a dynamic parameters.